tdrs-backend/tdpservice/clients.py

```diff
@@ -81,39 +80,3 @@ def scan_file(self, file, file_name) -> bool:
 
         logger.debug(f'Unable to scan file with name: {file_name}')
         return False
-
-
-def get_s3_client():
```

Comment:
Since we moved away from signed URLs this was only being used to satisfy tests. Those test assertions have been moved to `test_backends` instead.

Reply:
Additionally, if needed in the future a correctly scoped client can be accessed from the storage backend:
```
from tdpservice.backends import DataFilesS3Storage
client = DataFilesS3Storage().connection.meta.client
```